templates/threatconnect.hbs

```diff
@@ -982,56 +1088,183 @@
-                          <span class='info-bar-key'>Severity</span>
-                          <span class='info-bar-value {{case.__severityColor}}'>
-                            {{case.severity}}
-                          </span>
+
+                        <div class="info-bar-pill">
+                          <span class="info-bar-key">Severity</span>
+                          {{#if case.__isEditing}}
+                            <select
+                              class="info-bar-value"
+                              value={{or case.__newSeverity case.severity}}
+                              {{on "change" (fn (action "updateSeverity" case.id))}}
+                            >
+                              <option value="Critical">Critical</option>
+                              <option value="High">High</option>
+                              <option value="Medium">Medium</option>
+                              <option value="Low">Low</option>
+                            </select>
+                          {{else}}
+                            <span class="info-bar-value">{{case.severity}}</span>
+                          {{/if}}
                         </div>
-                        <div class='info-bar-pill'>
-                          <span class='info-bar-key'>Resolution</span>
-                          <span class='info-bar-value'>
-                            {{case.resolution}}
-                          </span>
+
+                        <div class="info-bar-pill">
+                          <span class="info-bar-key">Resolution</span>
+                          {{#if case.__isEditing}}
+                            <select
+                              class="info-bar-value"
+                              value={{or case.__newResolution case.resolution}}
+                              {{on "change" (fn (action "updateResolution" case.id))}}
+                            >
+                              <option value="Containment Achieved">Containment Achieved</option>
+                              <option value="Deferred / Delayed">Deferred / Delayed</option>
+                              <option value="Escalated">Escalated</option>
+                              <option value="False Positive">False Positive</option>
+                              <option value="In Progress / Investigating">In Progress / Investigating</option>
+                              <option value="Rejected">Rejected</option>
+                              <option value="Restoration Achieved">Restoration Achieved</option>
+                            </select>
+                          {{else}}
+                            <span class="info-bar-value">{{case.resolution}}</span>
+                          {{/if}}
                         </div>
+
+                        {{#if case.__isEditing}}
+                          <div class="info-bar-pill">
+                            <span class="info-bar-key">Attribute Types</span>
```

Comment:
The editing input for Attribute Types looks like it shoudl be moved down to it's own "row". Right now it's squished in with "status", "severity", and "resolution" making the values of all the drop down unreadable:
![image](https://github.com/user-attachments/assets/840ea547-5d17-4c2b-8109-2aaf327969b7)